frontend/components/Course/InstructorQueuePage/Queue.tsx

```diff
@@ -87,17 +186,25 @@ const Queue = (props: QueueProps) => {
                 mutate={mutateQuestions}
                 closeFunc={() => setClearModalOpen(false)}
             />
-            <Header as="h3">
-                {queue.name}
-                <Header.Subheader
-                    style={{
-                        whiteSpace: "break-spaces",
-                        wordBreak: "break-word",
-                    }}
-                >
-                    {queue.description}
-                </Header.Subheader>
-            </Header>
+            <div
+                style={{
+                    display: "flex",
+                    justifyContent: "space-between",
+                }}
+            >
+                <Header as="h3">
```

Comment:
Looks like something funky is going on with the spacing which wasn't before
![image](https://user-images.githubusercontent.com/52753462/150710044-f1cda642-3204-4b2e-af36-8340ae79d924.png)

I think -
before we had the title inside the grid, which provided spacing
now we're putting the title in a flexbox outside the grid, and need to add a margin? 

Not sure why it's not happening here though
![image](https://user-images.githubusercontent.com/52753462/150710613-8742c36d-cc71-441d-8ef2-aa0923611996.png)

Anyway just make it look good on chrome and we should be fine